tests/Kernel/Paragraphs/ParagraphsTestBase.php

```diff
@@ -45,7 +48,13 @@ protected function setUp() {
     $this->installEntitySchema('paragraph');
     $this->installEntitySchema('file');
     $this->installSchema('file', ['file_usage']);
-    $this->installConfig(['oe_paragraphs', 'filter']);
+    $this->installConfig([
```

Comment:
Same as before, lets do this in a dedicate class to not polute the rest of the tests.